medusa/config.py

```diff
@@ -294,6 +296,19 @@ def change_VERSION_NOTIFY(version_notify):
         app.version_check_scheduler.forceRun()
 
 
+def change_GIT_PATH():
```

Comment:
Yeah... not sure what the best solution for this would be as I followed the function naming pattern used by every other function in that file.